src/app/settings/views/Configuration/GeneralForm/ThemedRadioButton/ThemedRadioButton.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ReactNode } from "react";
+
+import classNames from "classnames";
+
+export type Props = {
+  checked?: boolean;
+  className?: string;
+  color?: string;
```

Comment:
Would be great if this was strongly typed. We do, in fact know that it can only be one in a set of defined colours. Anything else would be a bug. 

This could be done by defining an inline union type
`color?: "default" | "bark" | (...)`

Another option would be to refactor the array of colours you're passing down and using the imported type here.

```
enum ColorValue {
  PrussianGreen = "Prussian green",
(...)
}

const colors: { value: ColorValue; label: string }[] = [
  { label: "name", value: ColorValue.PrussianGreen },
(...)
];
```

Reply:
The above is just an example, You could also use more complex string type manipulation if you'd like, more on this here: https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html#uppercasestringtype

Reply:
Fixed this. I went for the enum for the color names, then in the form I map those onto radio buttons.